include/aspect/geometry_model/interface.h

```diff
@@ -144,6 +144,12 @@ namespace aspect
         virtual
         double height_above_reference_surface(const Point<dim> &position) const = 0;
 
+        /**
+         * Converts a Cartesian point into another coordinate system.
+         */
+        Point<dim>
+        cartesian_to_other_coordinates(const Point<dim> &position,
```

Comment:
So this is not a virtual function and actually doesn't use anything inside the current geometry model, right? Maybe move it into Utilities::Coordinates then?

Reply:
We had a discussion about this, but it uses the depth function of the geometry model.